setup/catalogs/data/bedrock.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: deployments.plural.sh/v1alpha1
+kind: PrAutomation
+metadata:
+  name: bedrock
+spec:
+  name: bedrock
+  icon: "https://github.com/pluralsh/plural-artifacts/blob/main/strapi/plural/icons/strapi.png" # placeholder, replace with a bedrock icon
+  identifier: mgmt
+  documentation: |
+    Sets up AWS Bedrock within Plural for your chosen cloud and cluster.
+  creates:
+    git:
+      ref: john/prod-3002-data-catalog-ai-catalog-bedrock-openai-gateway
+      folder: catalogs/data/bedrock
+    templates:
+      - source: helm
+        destination: helm/bedrock/{{ context.cluster }}
+        external: true
+      - source: terraform/{{ context.cloud }}
+        destination: "terraform/apps/bedrock/{{ context.cluster }}"
+        external: true
+      - source: bedrock-servicedeployment.yaml
+        destination: "bootstrap/apps/bedrock/{{ context.cluster }}/bedrock-servicedeployment.yaml"
+        external: true
+      - source: bedrock-stack.yaml
+        destination: "bootstrap/apps/bedrock/{{ context.cluster }}/bedrock-stack.yaml"
+        external: true
+      - source: README.md
+        destination: documentation/bedrock/README.md
+        external: true
+  repositoryRef:
+    name: scaffolds
+  catalogRef:
+    name: data-engineering
+  scmConnectionRef:
+    name: plural  # Adjust to your existing SCM connection
+  title: "Setting up Bedrock on cluster {{ context.cluster }} for {{ context.cloud }}"
+  message: |
+    Set up AWS Bedrock on {{ context.cluster }} ({{ context.cloud }})
+
+    This will provision IAM roles/policies for Bedrock, generate secrets,
+    and create ServiceDeployment + Ingress so that Plural can
+    route requests to AWS Bedrock.
+  configuration:
+    - name: cluster
+      type: STRING
+      documentation: Handle of the cluster you want to deploy bedrock to.
+    - name: stackCluster
+      type: STRING
+      documentation: Handle of the cluster used to run Infrastructure Stacks for provisioning the infrastructure. Defaults to the management cluster.
```

Comment:
let's be more precise with this, `Handle of the cluster used to run the stack to provision the bedrock IAM role, defaults to the management cluster.`